proprietary/design-tokens/src/imported/NLdoc/Ridderkerk.json

```diff
@@ -4,36 +4,36 @@
       "brand": {
         "default": {
           "parent": "NLdoc / Organisatie/Ridderkerk",
-          "type": "color",
-          "value": "#028532",
-          "description": ""
+          "description": "",
+          "$type": "color",
```

Comment:
@AlineNap Elke `type` en `value` hebben een `$` ervoor gekregen. Daardoor werkt het build script niet meer.
Heb jij iets aangepast in Tokens Studio?